src/settings/test.rs

```diff
@@ -6,31 +6,23 @@ use std::{
   collections::{HashMap, HashSet},
   env,
   error::Error,
-  sync::{LockResult, Mutex, MutexGuard},
 };
 
 use super::*;
 
-lazy_static! {
-  // HACK: mutex to prevent tests clobbering each other's environment variables
-  static ref ENVIRONMENT_MUTEX: Mutex<bool> = Mutex::new(true);
-}
```

Comment:
Because the tests are now run with `--test-threads=1`, I could delete this silly hack. (it wasn't preventing these tests from interfering with others that depend on `std::env`)